lib/src/screens/release_notes/release_notes_screen.dart

```diff
@@ -0,0 +1,152 @@
+                  fontFamily: 'Lato',
+                  color: Theme.of(context).accentTextTheme!.headline2!.backgroundColor!,
+                  ),
+              child: Row(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  Padding(
+                    padding: const EdgeInsets.only(right: 8.0),
+                    child: Text('•'),
+                  ),
+                  Expanded(
+                    child: Text(element),
+                  ),
+                ],
+              )),
+          SizedBox(
+            height: 16.0,
+          )
+        ],
+      ));
+    });
+    return notesWidgetList;
+  }
+
+  @override
+  void initState() {
+    super.initState();
+    getFileLines();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return AlertBackground(
+        child: Column(
+      crossAxisAlignment: CrossAxisAlignment.stretch,
+      children: [
+        Expanded(child: SizedBox()),
+        Expanded(
+          flex: 19,
+          child: Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 28.0),
+            child: Container(
+              decoration: BoxDecoration(
+                borderRadius: BorderRadius.circular(30.0),
+                gradient: LinearGradient(colors: [
+                  Theme.of(context).colorScheme.secondary,
+                  Theme.of(context).scaffoldBackgroundColor,
+                ], begin: Alignment.centerLeft, end: Alignment.centerRight),
+              ),
+              child: Padding(
+                padding: const EdgeInsets.symmetric(horizontal: 24.0),
+                child: Column(
+                  children: [
+                    Expanded(
+                      flex: 3,
+                      child: Container(
+                        alignment: Alignment.bottomCenter,
+                        child: DefaultTextStyle(
+                          style: TextStyle(
+                            decoration: TextDecoration.none,
+                            fontSize: 24.0,
+                            fontWeight: FontWeight.bold,
+                            fontFamily: 'Lato',
+                            color: Theme.of(context).accentTextTheme!.headline2!.backgroundColor!,
+                          ),
+                          child: Text(widget.title),
+                        ),
+                      ),
+                    ),
+                    Expanded(
+                      child: SizedBox(),
+                    ),
+                    Expanded(
+                      flex: 20,
+                      child: Container(
+                        child: ListView.builder(
+                          padding: const EdgeInsets.all(0),
+                          itemCount: notesWidgetList.length,
+                          itemBuilder: (BuildContext context, int index) {
+                            return notesWidgetList[index];
+                          },
```

Comment:
for future reference, it's better to use `FutureBuilder` when the UI is waiting for an async function, so that you can handle loading, and error states as well